devito/types/basic.py

```diff
@@ -919,25 +919,22 @@ def _data_alignment(self):
         """
         return default_allocator().guaranteed_alignment
 
-    def indexify(self, indices=None):
+    def indexify(self, indices=None, w_shift=False, extra_subs=None):
```

Comment:
Nitpicking, but `additional_subs` would be better than `extra_subs`?

Reply:
Doesn't fit in a line, why I made it shorter, and stil lthinks it explicit enough

Reply:
how about just "subs" (and "shift") ?